modules/swagger-parser/src/main/java/io/swagger/parser/util/SwaggerDeserializer.java

```diff
@@ -174,6 +176,176 @@ else if(!ROOT_KEYS.contains(key)) {
         return output;
     }
 
+    public static Optional<Boolean> allPathParametersAccountForExtractedPathParameters(final Swagger swagger) {
```

Comment:
Why use `Optional<Boolean>` instead of `boolean`?  We only have true/false for this operation.